lyaemu/spectra_mocking.py

```diff
@@ -0,0 +1,207 @@
+import numpy as np
+import fake_spectra
+from fake_spectra.plot_spectra import PlottingSpectra
+from fake_spectra.spectra import Spectra
+import matplotlib.pyplot as plt
+import array
+import h5py
+import seaborn as sn
+import os
+import subprocess
+
+def plot_pdf_Wiener_filtered(map_file=['60_512', '60_1024', '120_1024'],bins=np.arange(-1.0, 1.0, 0.02)):
+    
+    plt.figure(figsize=(10,8))
+    (mbin, hist) = get_pdf_Wiener_filtered(map_file = map_file, bins=bins)
+
+    for (j,i) in enumerate(map_file):
+
+        
+        plt.title('Convergence check')
+        plt.xlabel(r'$\delta_F$')
+        plt.ylabel(r'$ pdf \ of \ voxels, \ \Delta \delta_F \ = \ 0.02 $')
+
+        m = np.fromfile('./spectra/maps/map_'+i+'.dat')
+        if i == '60_512':
+            label = 'L = 60 h^-1 cMpc, Particles = 512'
+            ls = 'dashed'
+        if i == '60_1024':
+            label = 'L = 60 h^-1 cMpc, Particles = 1024'
+            ls = 'dotted'
+        if i == '120_1024':
+            label = 'L = 120 h^-1 cMpc, Particles = 1024'
+            ls = 'dashdot'
+        
+        #sn.distplot(m, bins=bins, hist=False, kde=True, label=label
+        plt.plot(mbin, hist[j], label=label, linestyle=ls)
+    plt.legend()
+    plt.savefig('hist_deltaF.png')
+
+
+def get_pdf_Wiener_filtered(bins=np.arange(-1.0, 1.0, 0.02)):
+    
+    
+    mbin = np.array([(bins[i]+bins[i+1])/2. for i in range(0,np.size(bins)-1)])
+    m = np.fromfile(map_file)
+    hist = np.histogram(m, bins=bins, density=True)[0]
```

Comment:
I'm confused by this: this is a histogram, not a Wiener filter. Did you intend to Wiener filter initially and then change your mind after seeing the results?